service/controller/chart/v1/resource/release/desired.go

```diff
@@ -0,0 +1,135 @@
+package release
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/chart-operator/service/controller"
+	"github.com/giantswarm/chart-operator/service/controller/chart/v1/key"
+)
+
```

Comment:
New chart CRD version is a lot leaner. No channels magic we just pull the tarball and parse its version with helmclient.